CRBM-Viz/src/app/Layout/links/links.component.ts

```diff
@@ -8,5 +9,13 @@ import { AuthService } from 'src/app/Services/auth0.service';
 export class LinksComponent implements OnInit {
   constructor(public auth: AuthService) {}
 
-  ngOnInit() {}
```

Comment:
As mentioned in #29 this component will be deleted, so it is not a good idea to have the profile load here. In any case, it is probably not a good idea to have the profile load in an unrelated component. Can this be moved to the main app component?